packages/js/browserinstance.js

```diff
@@ -725,8 +725,12 @@ class BrowserInstance {
         try {
             await this.driver.wait(async () => {
                 obj = await this.executeScript(function(titleOrUrl) {
+                    let isMatched: document.title.toLowerCase().indexOf(titleOrUrl.toLowerCase()) != -1 || window.location.href.indexOf(titleOrUrl) != -1;
```

Comment:
You mean `let isMatched =`?

Reply:
It was a typo I made when copy-pasting my code, sorry. It should be fixed.